Readme.md

```diff
@@ -604,6 +610,12 @@ The following tools are required to run [create-root-depls](scripts/create-root-
 
 * ruby
 
+## How to manage stemcell?
+
+The same stemcell is used for all deployments, it is defined in [shared-config.yml](docs/reference_dataset/template_repository/shared-config.yml)
+or in [private-config.yml](docs/reference_dataset/config_repository/private-config.yml), but the version is managed by 
+`root-deployment` using dedicated [<root-deployment>-versions.yml](docs/reference_dataset/template_repository/hello-world-root-depls/hello-world-root-depls-versions.yml)
+
```

Comment:
@o-orand thanks for the documentation update. 

Suggestion improvement below:

In order to be portable across multiple infrastructures, and allow for centralized stemcell upgrades, deployment authors rely on COA to manage stemcells including offline replication, uploading to bosh director and purge, and stemcell version selection at deployment time. Deployment authors therefore use the following syntax in their deployment manifests:

```yml 
stemcells:
- alias: trusty
  os: ubuntu-trusty
  version: latest
```

At deployment time, the deployment manifest is transformed by COA to load the expected stemcell version (say `3586.25`).

The same stemcell generation (currrently `ubuntu-trusty`) is used for all deployments as defined by authors in [shared-config.yml](docs/reference_dataset/template_repository/shared-config.yml)
or overloaded by operators in [private-config.yml](docs/reference_dataset/config_repository/private-config.yml).

The exact stemcell version (say `3586.25`) is used within a root-deployment as defined by authors in [\<root-deployment\>-versions.yml](docs/reference_dataset/template_repository/hello-world-root-depls/hello-world-root-depls-versions.yml)

Ability to support more than one stemcell (e.g. both `trusty` and `xenial) is considered in #155 
Ability to fetch conditionally fetch online stemcells is considered in #128
Ability to for authors to use different stemcell version per deployment is considered in #15

Reply:
Additionally the stemcell sections would be live in the `Template engine reference documentation` section than in the FAQ

Reply:
Thanks for the rewording, I will update the readme accordingly except the last part with issue reference. I'm not sure we need to reference issue in the readme. 
About private-config.yml, the overriding mechanism is already Unit Tested in [config-spec.rb](spec/lib/config_spec.rb) , I won't include more tests in reference_dataset to avoid a reverse test pyramid (ie more acceptance test than unit test).

Reply:
@gberche-orange as discussed I've included commented values for `private-config.yml`